README.md

```diff
@@ -0,0 +1,233 @@
+# Logtify
+
+[![CircleCI](https://circleci.com/gh/dial-once/node-logtify.svg?style=svg)](https://circleci.com/gh/dial-once/node-logtify)
+
+**[ES6]** Logger based on ``Chain of Responsibility`` pattern and ``winston`` module
+
+
+## Installing the module
+```bash
+npm i @dialonce/logtify
+```
+
+This module is an implementation of a chain, where each element is a small submodule, that does it's own small work with the same identical incoming data. ``Console`` chain element is available by default.
+
+Each chain element contains a ``nextChainLink`` property, which is a link to the next element of the chain. It also has a ``handle`` function, which accepts ``Message`` - a **frozen** message object and a ``link`` function, that is meant to set the ``nextChainLink``. A ``handle`` function processes the message and eventually passes is further along the chain to the next element (if exists).
+
+If needed, new chain links and adapters can be added to the chain.
+
+## Configuration
+### Minimal
+Basic usage exposes the chain and a winston adapter ``logger``:
+```js
+const { chain, logger } = require('@dialonce/logtify')();
+```
+### Parameters
+You can configure the chain by passing the following parameters or setting up the environment variables. Such settings will be passed to each chain element sequentially:
+```js
+const { chain, logger } = require('@dialonce/logtify')({
+    CONSOLE_LOGGING: true, // switches on the console link chain
+    MIN_LOG_LEVEL: 'info' // minimal message level to be logged
+});
+```
+**NOTE!** Environment variables have a higher priority over the settings object
+
+### Add new chain element
+By default, logtify contains only 1 chain link - `Console` chain link. However, you can add new chain links as the following:
+```js
+const { LogentriesChainLink } = require('@dialonce/logtify-logentries');
+const { BugsnagChainLink } = require('@dialonce/logtify-bugsnag');
+const { chain, logger } = require('@dialonce/logtify')({
```

Comment:
suggestion: From documentation standpoint maybe it would be best to focus on getting only logger and notifier objects and leave the chain out until the part of docs where you actually use it (and describe please that it is an array as the others are objects)